Makefile

```diff
@@ -49,7 +49,7 @@ test-runner: emake .FORCE
 	$(MAKE) -C CommandLine/testing/
 
 required-directories: .FORCE
-	mkdir -p "$(WORKDIR)"
-	mkdir -p "$(CODEGEN)Preprocessor_Environment_Editable/"
+	@mkdir -p "$(WORKDIR)"
```

Comment:
This could be the first thing to run. What if the filesystem is read-only and these fail?

Reply:
itll still print the error froim mkdir

[greg@greg-desktop shush-jdi]$ make required-directories
mkdir: cannot create directory ‘/derp/’: Permission denied
make: *** [Makefile:52: required-directories] Error 1

Reply:
Ah, good.